README.md

```diff
@@ -1,2 +1,50 @@
-# Microverse_capstone-1
-Capstone project from module-1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Capstone project from module-1
+
+## Built With
+
+- Major languages : HTML , CSS & JavaScript
+- Frameworks : none
+
+## Getting Started
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
+[Live Demo Link]()
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@yonas44](https://github.com/yonas44/Microverse_capstone-1)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-I want to thank [Creative_Commons](https://creativecommons.org/licenses/by-nc/4.0/) for allowing me to use their template in designing and building this website.
```

Comment:
The README should  contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).